cfme/common/host_views.py

```diff
@@ -98,7 +106,7 @@ class security_accordion(Accordion):  # noqa
     def is_displayed(self):
         return (
             self.in_compute_infrastructure_hosts and
-            self.breadcrumb.active_location == self.context["object"].expected_details_breadcrumb
+            self.context["object"].expected_details_breadcrumb in self.breadcrumb.active_location
```

Comment:
This was broken before. Since `self.breadcrumb.active_location` includes `hostname (Summary)` while  `self.context["object"].expected_details_breadcrumb` just had `hostname`

Reply:
Yeah that seems to have worked locally. Hope it does in PRT too.